common/source/docs/common-pixhawk-wiring-and-quick-start.rst

```diff
@@ -1,150 +1,157 @@
-
-.. figure:: ../../../images/FRSkyTaranis.jpg
-   :target: ../_images/FRSkyTaranis.jpg
-
-   FRSky Taranis Transmitter
-
+
+.. figure:: ../../../images/FRSkyTaranis.jpg
+   :target: ../_images/FRSkyTaranis.jpg
+
+   FRSky Taranis Transmitter
+
 Connect buzzer and safety switch
 ================================
-
+
 The buzzer and safety switch button are mandatory for Pixhawk. Connect
 to the BUZZER and SWITCH ports as shown.
-
-.. image:: ../../../images/pixhawk-required_peripherals.jpg
-    :target: ../_images/pixhawk-required_peripherals.jpg
-
-.. warning::
-
+
+.. image:: ../../../images/pixhawk-required_peripherals.jpg
+    :target: ../_images/pixhawk-required_peripherals.jpg
+
+.. warning::
+
    Mount the beeper is at least 5cm away from the flight
-   controller or the noise may upset the accelerometers.
-
+   controller or the noise may upset the accelerometers.
+
 3DR GPS+Compass
 ===============
-
+
 The :ref:`3DR UBlox GPS + Compass Module <common-installing-3dr-ublox-gps-compass-module>` is the
 recommended GPS for Pixhawk on ArduPilot.  The GPS ports are connected
 with the six-position DF13 cable, and the MAG port is connected to the
 I2C port with the four-position DF13 cable.
-
-.. figure:: ../../../images/GPS_TopAndSide.jpg
-   :target: ../_images/GPS_TopAndSide.jpg
-
-   3DR UBlox GPS + Compass Module
-
+
+.. figure:: ../../../images/GPS_TopAndSide.jpg
+   :target: ../_images/GPS_TopAndSide.jpg
+
+   3DR UBlox GPS + Compass Module
+
 The topic :ref:`3DR UBlox GPS + Compass Module <common-installing-3dr-ublox-gps-compass-module_connecting_to_pixhawk>`
 shows how to connect to Pixhawk and include additional configuration and
 mounting information.
-
+
 Connect Motors
 ==============
-
-.. image:: ../../../images/pixhawk_motor_outputs.jpg
-    :target: ../_images/pixhawk_motor_outputs.jpg
-
-[site wiki="copter"]
+
+.. image:: ../../../images/pixhawk_motor_outputs.jpg
+    :target: ../_images/pixhawk_motor_outputs.jpg
+
+[site wiki="copter"]
 For Copter see :ref:`Connect ESCs and Motors <copter:connect-escs-and-motors>`.
-
+
 In overview, for copters connect each signal wire from the PDB to the
 main output signal (S) pins by motor number:
-
+
 -  Pin 1 = Motor 1 - - Pin 5 = Motor 5
 -  Pin 2 = Motor 2 - - Pin 6 = Motor 6
 -  Pin 3 = Motor 3 - - Pin 7 = Motor 7
--  Pin 4 = Motor 4 - - Pin 8 = Motor 8
+-  Pin 4 = Motor 4 - - Pin 8 = Motor 8
 [/site]
-
-[site wiki="plane"]
+
+[site wiki="plane"]
 For planes connect the control channel wires to the main output signal
 pins:
-
+
 -  Pin 1 = Aileron
 -  Pin 2 = Elevator
 -  Pin 3 = Throttle
--  Pin 4 = Rudder
+-  Pin 4 = Rudder
 [/site]
-
-[site wiki="rover"]
+
+[site wiki="rover"]
 For Rovers connect the throttle and steering wires to the main output
 signal pins:
-
+
 -  Pin 3 = Throttle
--  Pin 1 = Steering
+-  Pin 1 = Steering
+
+The skid-steer parameters are used to configure vehicles that have fixed wheels and steer like

```

Comment:
Thanks HugesD and thanks Hamish for the review.  Its annoying that github inserts all these unnecessary changes - makes it useless really.
I'll update the PR and remove the unnecessary changes and then resubmit - it will still be under your authorship HughesD so you'll still get the credit :-)
Thanks, Grant.